cityofaustin/atd-vz-data
# Update local stack env var and replace Hasura metadata

## Associated issues
n/a

This PR addresses a warning in the Hasura console that happened after [updating the event triggers to use an environment variable](https://github.com/cityofaustin/atd-vz-data/pull/1205) called `HASURA_TRIGGER_API_KEY`. It also replaces the existing metadata that contains the leaked secret value with the updated metadata.

~_Edit: Also snuck in an update to gitignore the zappa_settings.json so we can prevent committing those secrets accidentally if we deploy the API manually._~ Nvm [Tilly handled this in another PR](https://github.com/cityofaustin/atd-vz-data/pull/1214) 🙏

## Testing
**URL to test:** 
Local only

**Steps to test:**
Spin up the local stack and go to the Hasura console. You should not be redirected to the settings page with the warning `You have been redirected because your GraphQL Engine metadata is in an inconsistent state`.

---
#### Ship list
